Orchestrator/docs/NLRModels.md

```diff
@@ -159,3 +157,4 @@ The models are released under the following [License Terms][6].
 [5]: https://github.com/snipsco/snips-nlu-metrics "Snips NLU Metrics"
 [6]: ./LICENSE.md "License agreement"
 [7]: https://en.wikipedia.org/wiki/Transformer_(machine_learning_model)
+[8]: https://github.com/pytorch/fairseq/tree/master/examples/xlmr#introduction
```

Comment:
[Ignore] Still wonder how we can provide attribution here - this is a direct point to xlmr leveraging their work.